scripts/install_third_party.sh

```diff
@@ -148,6 +148,14 @@ if [ ! -d "$TOOLS_DIR/isort-4.2.15" ]; then
   rm isort-4.2.15.tar.gz
 fi
 
+# Install GitPython.
+echo Checking if GitPython is installed in third_party
+if [ ! -d "$TOOLS_DIR/GitPython" ]; then
+  echo Installing GitPython
+
+  pip install gitpython --target="$TOOLS_DIR/GitPython"
```

Comment:
A couple of points here:
1. Can we fix the version of `gitpython` we use here?
2. Can we use curl instead of pip?

Also, the tests seem to be failing on Travis due to import error (see [here](https://travis-ci.org/oppia/oppia/jobs/473579310#L590)).

Reply:
done